packages/financial-templates-lib/src/logger/PagerDutyTransportV2.ts

```diff
@@ -0,0 +1,49 @@
+// This transport enables winston logging to send messages to pager duty v2 api.
```

Comment:
one thing this is missing is the ability to switch on `notificationPath` to send to different `serviceIds` as done in the "v1" pagerduty transport.https://github.com/UMAprotocol/protocol/blob/9fabd6cd7d2f1d0508199f779f64957f950c5f42/packages/financial-templates-lib/src/logger/PagerDutyTransport.ts#L42 this lets us define a config like this:

```json
"PAGER_DUTY_CONFIG": {
        "pdApiToken": "x",
        "fromEmail": "x",
        "defaultServiceId": "x",
        "customServices": {
          "risk-management": "x",
          "optimistic-oracle": "x",
          "dev-x": "x",
          "outcome-expirations": "x",
          "polymarket-notifier": "x"
        }
      },
```

and then logs go to each "customService" if the `notificationPath` is set. else, it goes to the `defaultServiceId`

Reply:
i have a better understanding of wahts going on with this, and have added a way to route to different services, using a similar mechanism as the v1 notifier